composer.json

```diff
@@ -30,8 +30,8 @@
         "doctrine/deprecations": "^0.5.3 || ^1",
         "doctrine/event-manager": "^1.2 || ^2.0",
         "psr/log": "^1.1.3 || ^2 || ^3",
-        "symfony/console": "^4.4.16 || ^5.4 || ^6.0",
-        "symfony/stopwatch": "^4.4 || ^5.4 || ^6.0",
+        "symfony/console": "^5.4 || ^6.0",
```

Comment:
Symfony 6.3 is the one with support (from 6.x line) right now so it can be bumped as well